src/style.css

```
@@ -0,0 +1,32 @@
+.text p {
+  margin: unset;
+  padding: 10px;
+  background-color: #ddd;
+  font-size: 16px;
+}
```

навешай класс text на `p`
```suggestion
.text {
  margin: unset;
  padding: 10px;
  background-color: #ddd;
  font-size: 16px;
}
```